docs/measure_scripts.md

```diff
@@ -0,0 +1,24 @@
+# Measure scripts
+
+FINESSE allows users to trigger a sequence of recordings with the mirror at different
+angles. The exact sequence, defined in terms of what the sequence of angles is and how
+many repeats to do, can be saved to a files, which we are calling "measure scripts".
+
+These files are written in [YAML](https://yaml.org/) format, with certain properties.
+Here is an example:
+
+```yaml
+count: 10
+sequence:
+- angle: zenith
+  count: 1
+- angle: 10.0
+  count: 3
+```
```

Comment:
I thought we have agreed to give a more informative name to the `count` keyword in the two contexts. See https://github.com/ImperialCollegeLondon/FINESSE/pull/63#discussion_r1043039325

Reply:
My bad. I'll do that now